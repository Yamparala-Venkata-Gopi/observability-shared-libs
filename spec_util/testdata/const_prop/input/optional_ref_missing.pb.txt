# MethodTemplate proto

method_id: {
  name: "new_method_with_refs"
}

arg_templates: {
  key: "arg1"
  value: {
    optional_template: {
      value_template: {
        ref: {
          method_index: 0
          arg_ref: {
            key: "primitive_constant_arg_missing"
            data_ref: {
              primitive_ref: {}
            }
          }
        }
      }
    }
  }
}
