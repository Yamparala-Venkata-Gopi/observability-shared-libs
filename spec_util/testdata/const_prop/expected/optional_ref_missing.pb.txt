# MethodTemplate proto

method_id: {
  name: "new_method_with_refs"
}
arg_templates: {
  key: "arg1"
  value: {
    value: {
      optional: {
        none: {}
      }
    }
  }
}
